### PR TITLE
Fix created logs are not added into processorLogs

### LINKED ISSUE #1281: LogProcessorFramework `removeLogProcessor`/`shutdown` not working
## Description
LogProcessorFramework has method **removeLogProcessor**, which takes the log identifier as the parameter. This however is neither stopping nor removing the log processor. 

Possible reason could be that the `Map processorLogs` in StandardLogProcessorFramework.java, which is being used to remove the logprocessor, was never actually populated while building the log processor. 

## How to reproduce?
* Add a log processor with some log processor identifier
* Do some operation in janusgraph to see that the log processor is functional
* remove the logprocessor using removeLogProcessor method
* Do a operation again to find out that the log processor was not removed

### PR DESCRIPTION
**Pull request by porunov**
Due to the stale #1742 PR I am opening a new PR to resolve requested changes. This PR is a copy paste of #1742 with resolved requested changes.

Fixes #1281
Suppresses #1742 

Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.janusgraph.diskstorage.util.time.TimestampProvider;
+import org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.janusgraph.graphdb.database.serialize.Serializer;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+import static org.easymock.EasyMock.*;
```
Comment (FlorianHockmann) [inline janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.easymock.EasyMock'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5115204)

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5115204&bid=16816766)

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (FlorianHockmann) [inline janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5115204)

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java (hunk 1 above)
Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: The import is used. Codacy not detecting it.

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java (hunk 2 above)
Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: The import is used. Codacy not detecting it.

Comment (farodin91) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Codacy says that some imports aren't used.

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.log;
+
+import org.easymock.EasyMockSupport;
```
Comment (farodin91) [inline janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: I'm not much a fan of easymock. I wouldn't this easy possible to do with mockito

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java (hunk 2 above)
Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: I don't know why it complains. assert functions are used in the code, so it is strange

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java (hunk 3 above)
Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Sure, I will change it to mockito

Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Changed to mockito

**Comment by FlorianHockmann** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5115204&bid=16816766)

File: janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import org.janusgraph.graphdb.database.serialize.Serializer;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.LinkedList;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.*;
```
Comment (FlorianHockmann) [inline janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=5115204)

Comment (porunov) [reply janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java]: when, mock, anyString is used by this import. Cidacy do not detect it

**Pull request merged**